doc/md/writing-motoko/async-data.md

```diff
@@ -69,6 +70,26 @@ Unlike a local function call, which blocks the caller until the callee has retur
 
 Awaiting a future a second time will just produce the same result, including re-throwing any error stored in the future. Suspension occurs even if the future is already complete; this ensures state changes and message sends prior to every `await` are committed.
 
+## Using parentheticals to modify message send modalities
+
+In the above examples all messages sent to the `Counter` actor are of unbounded-wait kind and attach no cycles. Both of these
+aspects can be changed by syntactically adding a parenthetical and thus modifying the modality of the send.
+
+To add cycles to the send one would write
+``` motoko no-repl
+let a = (with cycles = 42_000_000) Counter.bump();
+```
+Similarly, one can specify a cutoff for potentially long-running messaging by providing a timeout, and thus delivering a bounded-wait
+message to the destination actor
```

Comment:
Similarly, one can specify an explicit timeout to apply when awaiting the result of the message. This is useful in applications that prefer best-effort  over guaranteed response message delivery: